contrib/babelfishpg_common/src/varchar.c

```diff
@@ -1246,7 +1241,7 @@ float82bpchar(PG_FUNCTION_ARGS)
 	int maxlen = (typmod == -1) ? 30 : (typmod - VARHDRSZ);
 	Datum res;
 	/* 32 length as double_to_shortest_decimal_buf always returns string with length less that 30*/
-	char	   *ascii = (char *) palloc0(32);
+	char	   *result;
```

Comment:
NIT: Remove comment related to `double_to_shortest_decimal_buf`, since we are not using that anymore.

Reply:
Removed